test/integration/targets/filesystem/defaults/main.yml

```diff
@@ -11,4 +11,8 @@ tested_filesystems:
   ext2: {fssize: 10, grow: True}
   xfs: {fssize: 20, grow: False}  # grow requires a mounted filesystem
   btrfs: {fssize: 100, grow: False}  # grow not implemented
+  vfat:
+    fssize: 20
+    grow: False  # fatresize < 1.0.4 has segfaults
+    opts: "{{ '-F12' if ansible_system == 'FreeBSD' else ''}}"
```

Comment:
Is `-F12` parameter mandatory ?

Reply:
I'm not 100% sure what's necessary on FreeBSD, I was just copying the test command here since it doesn't seem to work without any options. `-F12` is also not enough, it complains about a missing label - I'm not sure why the command in `overwrite_another_fs.yml` works and this doesn't?